web/src/hooks/useScript.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+
+const useScript = (url: string): void => {
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = url
+    script.async = true
+
```

Comment:
maybe its safer to also add here
`script.defer = true`

https://javascript.info/script-async-defer#defer

Reply:
Good point. Thanks for sharing!

Reply:
Why is async safer then defer?

Reply:
Both defer and async does not make sense. 

Defer and Async both download the script while HTML rendering (not waiting with rendering while downloading)
Defer executes script after HTML rendering is complete. Async waits with HTML rendering while script executing. 

See a nice image for explanation here: https://stackoverflow.com/questions/10808109/script-tag-async-defer

I decided to use defer instead of  async, as we do not what this chat-bot to block/break anything.